resources/references/app-reference/webhook-events-reference.md

```diff
@@ -0,0 +1,81 @@
+# Webhook Event Reference
+
+| Event | Description | Permissions needed | Payload
+| :--- | :--- | :--- | :--- |
+|`checkout.customer.before.login` | Triggers as soon as a customer logs in | - | {&quot;email&quot;:&quot;string&quot;}
+|`checkout.customer.changed-payment-method` | Triggers when a customer changes his payment method in the checkout process | `customer:read` | {&quot;entity&quot;:&quot;customer&quot;}
```

Comment:
the formatting seems off right? we should not need `&quot;` right? `"` should also work, or was this intended?